src/coreclr/utilcode/longfilepathwrappers.cpp

```diff
@@ -54,14 +54,15 @@ LoadLibraryExWrapper(
 
     EX_TRY
     {
-
         LongPathString path(LongPathString::Literal, lpLibFileName);
 
         if (LongFile::IsPathNotFullyQualified(path) || SUCCEEDED(LongFile::NormalizePath(path)))
         {
 #ifdef HOST_WINDOWS
-            //Adding the assert to ensure relative paths which are not just filenames are not used for LoadLibrary Calls
-            _ASSERTE(!LongFile::IsPathNotFullyQualified(path) || !LongFile::ContainsDirectorySeparator(path));
+            // Ensure relative paths which are not just filenames are not used for LoadLibrary calls.
```

Comment:
Is this a breaking change? Are there cases that worked just fine in release builds before this change and that are going to fail with exception now?

Reply:
I don't think so, but that is a concern. I believe the tricky bit here was having relative path that couldn't be resolved to a real path anyways. The inner message might be different but the public facing exception type should remain the same.

Reply:
I think this is breaking change. I have done some ad-hoc testing and the relative paths worked just fine before this change as far as I can tell.

For example, try `NativeLibrary.Load(@"..\windows.x64.Release\clrjit_win_x86_x64.dll")`, with `clrjit_win_x86_x64.dll` in appropriate place.

Delete the assert and regression test instead?